src/pages/[id]/index.js

```diff
@@ -20,6 +21,21 @@ const MemberProfile = ({
   tasks,
   errorMessage,
 }) => {
+  const [activeTasksData, setActiveTasksData] = useState([tasks]);
+  const getId = () => {
+    const router = useRouter();
+    const { id } = router.query;
+    return { id };
```

Comment:
Why are we returning the Id as an object? 🤔

Reply:
Actually I was a bit confused and thought we should return an object here...so we shouldn't be returning an object right??

Reply:
Since here, you are using `getId()` at `getActiveTasksURL()` function which requires `id` as argument and not `{ id }`. So we should return only id. @sumitd94 correct me if I am wrong.

Reply:
We shouldn't be returning an object here, just returning the id should work. Did you test whether it works like this? 🤔 
Let's return it as normal id without object
@Ishikabaid

Reply:
It wasn't giving any error in both the cases...Anyways I've returned Id without object.